### ISSUE TITLE
Blank screen encountered after using Kedro Viz.

### ISSUE THREAD

**Issue by Kale-S** [kedro-org/kedro-viz#1500]
## Description
Initially, I attempted to execute the spaceflights example provided in the documentation. However, whenever I tried launching the Kedro Viz interface, it opened the local host at 127.0.0.1:4141, but all I could see was a blank page.

Moreover, I encountered a similar issue with my own Kedro project. Interestingly, when I ran the project on a different machine, it functioned correctly

## Steps to Reproduce
Its hard to reproduce because its only happend on my working machine. 

## Actual Result
When I execute the `%run_viz` command within the Jupyter Lab using the Kedro kernel, I encounter a white page displaying the message "127.0.0.1 has refused the connection."

## Your Environment
I used the src/requirement.txt as Environment
 
* Kedro version used  v0.18.12:
* Kedro viz version used 6.4.0:
* Python version used 3.10.12:
* Operating system and version Windows 11:

**Comment by tynandebold** [comment]
Hey @Kale-S, thanks for the issue.

Could you please provide more context in order for us to help you better? It would be great to see if there are any errors in the browser console and also to know what type of machine you're working on (Windows/Mac/etc.) and what browser and browser version.

**Comment by johnkmaxi** [comment]
I am having the same issue. When I run `kedro viz` the browser window just shows a blank webpage.

![image](https://github.com/kedro-org/kedro-viz/assets/14312461/7c56a302-5973-4cf3-a3e7-9f4527348911)

terminal output:
![image](https://github.com/kedro-org/kedro-viz/assets/14312461/5967bd58-58c3-48f5-a0f3-d0783ea94314)

**Comment by tynandebold** [comment]
@johnkmaxi could you please provide a minimum viable repo so we can try to reproduce this on our end? That will help us debug faster.

**Comment by tynandebold** [comment]
@Kale-S we merged a PR (#1576) and released a version that might solve this. Could you please try out Kedro-Viz 6.6.1 and see if this solves your issue?

**Comment by Fatafim** [comment]
For me on versions: kedro-0.18.14 kedro-viz-6.6.1, nothing's changed, still blank page. Not sure if it's useful but the browser returns js erorrs.

![image](https://github.com/kedro-org/kedro-viz/assets/57731567/2605e467-0bef-47e5-9783-e4a4e51327fa)

**Comment by MehdiNV** [comment]
Hi @Kale-S, @johnkmaxi - could you please execute `node -v` in your terminal and pass its output here?

I'm curious on the `WinHttpSyncHttpClient Failed setting secure crypto protocols with error code 87` shown in the outputs earlier, wondering whether the node version could be related as it can sometimes mess up SSL connections on non-Windows operating systems.

**Comment by astrojuanlu** [comment]
A user with this problem https://stackoverflow.com/q/77370743/554319

**Comment by Fatafim** [comment]
@astrojuanlu 

> A user with this problem https://stackoverflow.com/q/77370743/554319

That's the same user :)

> For me on versions: kedro-0.18.14 kedro-viz-6.6.1, nothing's changed, still blank page. Not sure if it's useful but the browser returns js erorrs.
> 
> ![image](https://user-images.githubusercontent.com/57731567/277749128-2605e467-0bef-47e5-9783-e4a4e51327fa.png)
> 
> EDIT: Interstingly, I was able to run it with WSL. It still doesn't work on Windows 10 though.

**Comment by MehdiNV** [comment]
Hi @Fatafim similarly could you run `node -v` and post the results? Curious to see npm / node versions

**Comment by Fatafim** [comment]
@MehdiNV Sure, here it is:
![image](https://github.com/kedro-org/kedro-viz/assets/57731567/0fafbf8b-4667-4653-89ad-ce739e6df6a9)

**Comment by MehdiNV** [comment]
> @MehdiNV Sure, here it is: ![image](https://user-images.githubusercontent.com/57731567/280484063-0fafbf8b-4667-4653-89ad-ce739e6df6a9.png)

Cheers! If you use a lower node version (~16 and below, e.g. 16.20.2), does anything change?

**Comment by Fatafim** [comment]
> > @MehdiNV Sure, here it is: ![image](https://user-images.githubusercontent.com/57731567/280484063-0fafbf8b-4667-4653-89ad-ce739e6df6a9.png)
> 
> Cheers! If you use a lower node version (~16 and below, e.g. 16.20.2), does anything change?

@MehdiNV Having downgraded node to 
![image](https://github.com/kedro-org/kedro-viz/assets/57731567/548d2b7f-69d0-40e4-a429-bba1a8f1c6d8)
nothing appears to have changed, still at: 
![image](https://github.com/kedro-org/kedro-viz/assets/57731567/e0f354c5-7b7b-4777-8be0-daba36c07d9b)

**Comment by MehdiNV** [comment]
@Fatafim Cheers for sharing - that's strange to see. If possible, would you be able to share a minimum viable repo? I'm struggling to reproduce this bug, and a repo close to yours would be massively useful hopefully for this

**Comment by Fatafim** [comment]
Hello,

I think I've solved the problem! Assuming that you have not made any big changes in kedro viz, it was a dependency conflict. Which one exactly? I do not know, but I realized it might have been the problem when I saw another conflict between kedro and icecream - which did NOT occur on another pc that had the same specification but it was installed solely for the project I had been using. So, **if you're having the same issue, reinstall you python entirely, including manual delete of all libraries.**

If any of you also has a problem with GridSearchCV with n_jobs>1 - **the solution stays the same.** 

@MehdiNV - so no wonder you couldn't reproduce it. I couldn't reproduce it on a virtual environment I was going to share with you either :)
